qpsolvers_benchmark/benchmark.py

```diff
@@ -95,6 +78,9 @@ def parse_command_line_arguments():
         "settings",
         help='settings to compare solvers on (e.g. "high_accuracy")',
     )
+    parser_plot.add_argument(
+        "--results-file", help="path to the results CSV file"
```

Comment:
Suggestion:
```suggestion
        "--results-file", help="path to the CSV file where results will be stored"
```

Reply:
This one is for the plot command (results are already stored), I suggest : "path to the CSV file where results are stored", let me know what you think.

Reply:
Sounds good :+1: